### PR TITLE
Refactor/nuxt update and netlify dev

### PR DESCRIPTION
**Pull request by marcoeh**
Adds #3

To do:
- [x] Update readme

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -13,57 +13,46 @@ module.exports = {
...
-    baseURL: '/'
-  },
+   ** Modules
+   */
+  modules: ['@nuxtjs/axios'],
 
   /*
-  ** Proxy config
-  */
-  proxy: {
```
Comment (mornir) [inline nuxt.config.js]: Actually I wanted to keep a proxy for localhost:3000. Netlify Dev serves by default the app at both 3000 and 8888. But I guess it's not really needed: if a developer wants to test the functions, he should visit the app on localhost:8888.

File: package.json [hunk 2]
```diff
@@ -13,31 +13,24 @@
   },
   "dependencies": {
     "@nuxtjs/axios": "^5.3.6",
-    "@nuxtjs/proxy": "^1.3.1",
     "dotenv": "^6.2.0",
     "element-ui": "^2.5.4",
-    "mailgun-js": "^0.22.0",
+    "encoding": "^0.1.12",
```
Comment (mornir) [inline package.json]: I can't see where this package ("encoding") is used. Is it a dependency from another package?

Comment (marcoeh) [reply package.json]: Yeah, it's because of [this issue with `node-fetch`](https://github.com/netlify/netlify-lambda/issues/142#issuecomment-483880089)

**Comment by marcoeh** [comment]
Thanks for updating the readme, I forgot about that.

**Pull request merged**